src/constants.js

```diff
@@ -60,6 +60,7 @@ export const STORM_CHANCE = 0.5
 
 export const SCARECROW_ITEM_ID = 'scarecrow'
 export const SPRINKLER_ITEM_ID = 'sprinkler'
+export const MINED_ITEM_ID = 'mined'
```

Comment:
I think we want get rid of this and all of its references (like in `items.js`). The idea with these constants is to associate specific items to their game logic. This pattern has become less useful over time and may not be terribly helpful anymore. In any case, I don't think we want to use or extend it for this PR.